app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
```

Comment:
**Optional**
- I highly recommend styling the part of signing up and logging in on the app for a better look and consistency with the rest of the design.
![Screenshot (2)](https://user-images.githubusercontent.com/47625407/183249943-cf8c3907-3d4c-4a30-9e55-ad580fbad44d.png)